src/ws_lowlevel_peer.rs

```diff
@@ -46,22 +42,18 @@ Example: TODO
 
 #[derive(Debug, Clone)]
 pub struct WsLlServer<T: Specifier>(pub T);
-impl<T:Specifier> Specifier for WsLlServer<T> {
+impl<T: Specifier> Specifier for WsLlServer<T> {
     fn construct(&self, p: ConstructParams) -> PeerConstructor {
         let inner = self.0.construct(p.clone());
         let opts = p.program_options;
-        inner.map(move |q, _| get_ws_lowlevel_peer(
-            WsLlContext::Server,
-            q,
-            opts.clone(),
-        ))
+        inner.map(move |q, _| get_ws_lowlevel_peer(WsLlContext::Server, q, opts.clone()))
```

Comment:
Aren't those `get_*_peer` function calls better be all multiline?

Reply:
I'd say "yes". These specific semantics don't quite match with rustfmt's, sadly :(